go/pkg/functions/gatekeeper/validate.go

```diff
@@ -47,7 +47,7 @@ func init() {
 	}
 }
 
-func createClient() (opaclient.Client, error) {
+func createClient() (*opaclient.Client, error) {
```

Comment:
If we change the return value here then shouldn't we also change how we handle the result of createClient in the Validate func?

Reply:
We should be good. What changed here was that the constraint framework went from returning an interface to the object itself (to comply with the guideline that interfaces should be defined by the consumer). The set of method signatures were stable and the same errors apply.